R/pkg/R/functions.R

```diff
@@ -1730,24 +1730,82 @@ setMethod("toRadians",
 
 #' to_date
 #'
-#' Converts the column into DateType.
+#' Converts the column into DateType, optionally with a format. Returns null if input is invalid.
```

Comment:
Take a look at the next push, I think that this will resolve it. Because this is not an action I don't think it's relevant to define what will happen when the data is collected. That is already documented according to how the types line up with one another.